.github/workflows/ci.yml

```diff
@@ -38,11 +39,9 @@ jobs:
     - uses: actions/checkout@v4
       with:
         submodules: true
-    - uses: actions-rs/toolchain@v1
+    - uses: dtolnay/rust-toolchain@stable
       with:
-        toolchain: stable
         components: rustfmt
-        profile: minimal
     - uses: arduino/setup-protoc@v3
     - uses: actions-rs/cargo@v1
```

Comment:
Here...

Reply:
I would recommend that we bring back this use of `actions-rs/cargo@v1` to get the test working again and then after this is merged we can circle back to solve this one in a separate PR.

Reply:
@lalitb I'd agree

Reply:
@cclauss do you still see any issue here? The unrelated CI failure was fixed by #1658 .